repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-link-document.yaml

```diff
@@ -0,0 +1,10 @@
+definitions:
```

Comment:
Could we get rid of `new-link-document` template query as it became a compound type now ?